### PR TITLE
Bugix/evcouplings 0.2

### PR DESCRIPTION
**Pull request by aggreen**
Fixed three bugs for release 0.2:

paralogs properly identified in best reciprocal hit concatenation
hmmsearch call in complexes pipeline has correct raw fasta alignment input
read_species_annotation_file chooses annotation column with more non-null values

### CODE REVIEW

**Comment by aggreen** [comment]
This branch passed testing on o2 for monomer and complex runs. Ready for release 0.2 once merged.

File: evcouplings/complex/similarity.py [hunk 1]
```diff
@@ -42,18 +41,18 @@ def read_species_annotation_table(annotation_file):
...
+        if sum(data[column].notnull()) > sum(data[annotation_column].notnull()):
             # use that column to extract data
             annotation_column = column
-            break
 
     # creates a new column called species with the species annotations
     data.loc[:, "species"] = data.loc[:, annotation_column]
 
-    return data
+    return data[["id","name","species"]]
```
Comment (thomashopf) [inline evcouplings/complex/similarity.py]: PEP8 ;)

Comment (aggreen) [reply evcouplings/complex/similarity.py]: every. single. time.

**Comment by aggreen** [comment]
Requested changes made, so I'm merging this pull request. We are good to go on evcouplings 0.2.

**Comment by sacdallago** [comment]
Thanks @aggreen :)

@b-schubert @thomashopf should we roll this out? In case: I can put it on my todo for next tue :)

**Comment by aggreen** [comment]
Tests passed so we are ready for rollout if you all agree

**Comment by thomashopf** [comment]
:thumbsup: 

Side note: the automatic deployment to pypi doesn't work for unknown reasons, so that would need fixing or manual uploading to pypi.

**Comment by sacdallago** [comment]
Alight, on my todo for tomorrow.

@thomashopf thanks for the heads up. It was set up via Travis, right? I'll try to research a fix and automate it again :)

**Pull request merged**